microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools-100.md

```diff
@@ -0,0 +1,136 @@
+
+For the history of this feature, see [Outlook: Add column in the Network activity log to track why resources are not loaded from DiskCache when ServiceWorker is present · Issue #16](https://github.com/MicrosoftEdge/DevTools/issues/16).
+
+
+<!-- ====================================================================== -->
+## When importing a performance profile, links now map to your original code
+
+<!-- Title: Use sourcemaps from Azure Artifacts symbol server to better debug performance issues -->
+<!-- Subtitle: Links from an imported performance profile now map to your original code because of source maps. -->
+
+When recording a performance profile, the flame chart in the **Performance** tool displays links from each event on the main thread to the corresponding JavaScript function in the **Sources** tool.  However, these links are broken when you export and import a performance profile.
+
+In Microsoft Edge 100, the links from an imported performance profile now use source maps from Azure Artifacts symbol server to map back to your familiar, original source code.
+
+![Links from imported performance profile use source maps go to your familiar, original source code.](devtools-100-images/links-perf-profile-orig-source-code.png)
+
+To learn more about using source maps in DevTools, see [Map the processed code to your original source code, for debugging](../../../javascript/source-maps.md).
+
+To get started with storing your source maps in Azure Artifacts symbol server and connecting to them from DevTools, see:
+*  [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md).
+*  [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md).
+
+
+<!-- ====================================================================== -->
+## Fix: The Waterfall view in the Network tool now matches Visual Studio Code themes
+
+<!-- Title: Themes from Visual Studio Code now apply to the Waterfall view -->
+<!-- Subtitle: The Waterfall view of requests in the Network tool now match the VS Code themes. -->
+
+In previous versions of Microsoft Edge, the **Waterfall** view of requests in the **Network** tool didn't match the themes from Visual Studio Code that apply to the rest of DevTools.  In Microsoft Edge 100, this issue has been fixed.
+
+The selected theme wasn't applied in the Waterfall view:
+
+![The selected theme wasn't applied in the Waterfall view.](devtools-100-images/waterfall-view-requests-network-no-theme.png)
+
+The selected theme is now applied to the Waterfall view of requests in the Network tool:
```

Comment:
Same thing as my previous comment. Can we add some additional information to this sentence before the image:
```suggestion
Now in Microsoft Edge 100, the selected theme is now applied to the Waterfall view of requests in the Network tool:
```